net/netatalk4/Portfile

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           meson 1.0
+
+name                netatalk4
+conflicts           netatalk
+
+github.setup        Netatalk netatalk netatalk-4-1-2
```

Comment:
this would result in the `version` to be set as "netatalk-4-1-2", which is clearly not what we want. Upstream names its releases/tags in a bit of an unconventional way and I am not sure what the best way is to deal with that. 
Probably the easiest is to explicitly set:
`version 4.1.2` on a separate line underneath the `github.setup` line.

Reply:
okay, so this should actually be changed to the following below. It will use the `tar.xz` from the upstream released version and also have then all other entries correct:
```
github.setup        Netatalk netatalk 4-1-2 netatalk-
version             [string map {- .} ${github.version}]
github.tarball_from releases
distname            netatalk-${version}
use_xz              yes
revision            0
```

Reply:
This is a better solution. I have updated to this code.